Makefile

```diff
@@ -1,16 +1,23 @@
+.POSIX:
+
 # Gets all packages that include a Rust crates
 WORKSPACES_WITH_RUST_MODULES := $(wildcard $(addsuffix /crates, $(wildcard ./packages/*)))
 
 # Gets all individual crates such that they can get built
 CRATES := $(foreach crate,${WORKSPACES_WITH_RUST_MODULES},$(dir $(wildcard $(crate)/*/Cargo.toml)))
 
-.POSIX:
+# add local Cargo install path and use it as custom shell PATH
+PATH := ${PATH}:${CURDIR}/.cargo/bin
+SHELL := env PATH=$(PATH) $(shell which bash)
+
+# use custom Cargo config file for each invocation
+cargo := cargo --config ${CURDIR}/.cargo/config.toml
 
 all: help
 
 .PHONY: $(CRATES) ## builds all Rust crates
 $(CRATES):
-# --out-dir is relative to working directory
+	# --out-dir is relative to working directory
```

Comment:
This will print `# --out-dir is relative to working directory` to the shell - whereas when removing the leading tab - `make` will consider this a comment

Reply:
moved back